packages/block-directory/src/store/actions.js

```diff
@@ -146,3 +150,54 @@ export function removeInstalledBlockType( item ) {
 		item,
 	};
 }
+
+/**
+ * Returns an action object used to indicate install in progress
+ *
+ */
+export function fetchInstallBlock() {
```

Comment:
The naming of `fetchInstallBlock` and `receiveInstallBlock` confuses me. The usual convention in Gutenberg is that dispatching a `fetchEntity` functions makes a network request and `receiveEntity` functions store a fetched entity in the data store. So when I first saw these function names I thought that `InstallBlock` was an entity.

Since as far as I can tell this action only toggles `isInstalling`, I'd suggest keeping it simple and changing these functions to `setIsInstalling( isInstalling )`. Thoughts?